### ISSUE TITLE
Feature Request: List of projects that use your API

### ISSUE THREAD

**Issue by bedwardly-down** [PLhery/node-twitter-api-v2#88]
Last night, I ported the twitter integration code for a Twitch bot I use from the standard twitter nodejs library to yours and found the transition to be mostly painless. Great job, and I’d love to see more projects move to your library. Twitter V2 is coming in full at some point meaning that the old library just won’t cut it long term.

The bot is called SogeBot, the developer is super friendly and has put a substantial amount of work into it. I’m still fairly new to Javascript and Typescript but plan on doing more with your library.

The PR for my port: https://github.com/sogehige/sogeBot/pull/4827

Thanks again. :)

**Comment by deadcoder0904** [comment]
Of course, this needs to be manually curated like [awesome](https://awesome.re) but here's a list if you're looking for inspiration → https://github.com/PLhery/node-twitter-api-v2/network/dependents?package_id=UGFja2FnZS0xNDMwMzYzNzI0

I sometimes use [Sourcegraph](https://sourcegraph.com/) with query `"twitter-api-v2":` → https://sourcegraph.com/search?q=context:global+%22twitter-api-v2%22:&patternType=literal

**Comment by bedwardly-down** [comment]
I didn’t know either of those resources existed. That’s amazing and I wonder if maybe that could be implemented into the current CI or git hooks or something pretty easily. It’d give more exposure for sure and would hopefully stop new projects from using the old library
